routes/v0/cases.js

```diff
@@ -20,7 +39,11 @@ casesRouter.post('/', async (req, res) => {
   res.json(createdCase);
 });
 
-casesRouter.put('/:id', async (req, res) => {
+/**
+ * We use asyncHandler(fn) here because expressJS expects the middleware to be a synchronous function,
+ * but editCasePermissions(args) is asynchronous
+ * */
+casesRouter.put('/:id', asyncHandler(editCasePermissions), async (req, res) => {
```

Comment:
It would be great to have some sort of default that blocks everything, and then we have to add opt-in permissions.  Just so that if we forget to add permissions, we don't open up too much.  I don't really see how to do that with this framework, though.